contrib/qpl-cmake/CMakeLists.txt

```diff
@@ -713,7 +713,8 @@ target_compile_definitions(_qpl
 
 target_link_libraries(_qpl
         PRIVATE ch_contrib::accel-config
-        PRIVATE ch_contrib::isal)
+        PRIVATE ch_contrib::isal
+        PRIVATE ch_contrib::qat)
```

Comment:
This is because they have duplicate symbol: crc32_gzip_refl_base, but now i find it can be covered by ch_contrib::isal library, so we don't need do this anymore

Reply:
> I believe the "official" name is `QATlib`. Could we name the library in CMake accordingly `qatlib`?

done

Reply:
If there is a dependency on the existing isa-l lib, then it would be nice if you could add `# requires: isa-l` behind `add_contrib (qatlib-cmake qatlib)` in contrib/CMakeLists.txt

Reply:
Done